builds/azure-pipelines/build-release.yml

```diff
@@ -22,7 +22,7 @@ schedules:
 variables:
   solution: '**/*.sln'
   configuration: 'Release'
-  versionMajor: 2
+  versionMajor: 3
```

Comment:
Are we keeping the version as 3 (but as preview) then? That should be fine, but it'd be good to go through after and make sure that all the versions are set up correctly.